how-i-can/news
# PER-5 Bottom Nav bar home button - For Shirley

**Test Plan**
- Open perch
- Note the default list of articles displayed
- Search for some articles
- Click home button
- Confirm the list of articles is reset to the default list

**The search string is not cleared. We will address that outside of this work.
